src/schema/v2/partner/partner.ts

```diff
@@ -414,6 +419,10 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             args
           )
 
+          console.log("IVE BEEN HIT")
+
+          console.log(args)
```

Comment:
PSA - if you start metaphysics from within VSCode terminal you can use breakpoints directly in vscode. It works really perfectly